### PR TITLE
PPC: temporary set quote active to true for third party plugins

### PR DESCRIPTION
**Pull request by vitaliyreznikov**
Fixes: https://app.asana.com/0/0/1161039988578263/f

# Type of change

- [ ] Bug fix (change which fixes an issue)
- [ ] New feature (change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?
Please validate that you have tested your change in at least one of the following areas:

- [ ] Successfully tested locally (or docker image)
- [ ] Successfully tested on a staging or sandbox server
- [ ] Successfully tested on a merchant's staging server

# Checklist:

- [ ] My code follows the style guidelines of this project.
- [ ] I have performed a self-review of my own code.
- [ ] I have commented my code, particularly in hard-to-understand areas.
- [ ] New and existing unit tests pass locally with my changes.
- [ ] I have created or modified unit tests to sufficiently cover my changes.
- [ ] I have added my Asana task link and provided a changelog message if applicable.

### CODE REVIEW

File: Helper/Order.php [hunk 1]
```diff
@@ -1046,11 +1046,21 @@ public function getExistingOrder($orderIncrementId)
...
+            $quote->setIsActive(true);
+        }
         $this->_eventManager->dispatch(
             'sales_quote_save_after', [
                 'quote' => $quote
             ]
         );
+        if (!$is_quote_active) {
+            $quote->setIsActive(false);
+        }
```
Comment (danac-gs) [inline Helper/Order.php]: How about `$quote->setIsActive($is_quote_active);`, no `if`?

File: Helper/Order.php [hunk 2]
```diff
@@ -1046,11 +1046,21 @@ public function getExistingOrder($orderIncrementId)
     protected function quoteAfterChange($quote)
     {
         $quote->setUpdatedAt($this->date->gmtDate());
+        // If it's PPC quote make it temporary active
+        // for third party plugins work
+        $is_quote_active = $quote->getIsActive();
+        if (!$is_quote_active) {
+            $quote->setIsActive(true);
+        }
```
Comment (danac-gs) [inline Helper/Order.php]: How about just `$quote->setIsActive(true);`, without the condition?

Comment (vitaliyreznikov) [reply Helper/Order.php]: I don't know, I'd prefer to save the condition if you don't mind.

**Comment by danac-gs** [comment]
@vitaliyreznikov please fix conflicts

File: Helper/Order.php [hunk 3]
```diff
@@ -1040,11 +1040,21 @@ public function getExistingOrder($orderIncrementId)
     protected function quoteAfterChange($quote)
     {
         $quote->setUpdatedAt($this->date->gmtDate());
+        // If it's PPC quote make it temporary active
+        // for third party plugins work
+        $is_quote_active = $quote->getIsActive();
```
Comment (danac-gs) [inline Helper/Order.php]: We should use camelCase for variables, but lets leave it like this for now and merge it.

**Pull request merged**